cformers/interface.py

```diff
@@ -65,6 +65,9 @@ def get_modes(self):
     'bigscience/bloom-7b1': ModelUrlMap(
         cpp_model_name="bloom",
         int4_fixed_zero="https://huggingface.co/ayushk4/bigscience-.-bloom-7b1/resolve/main/int4_fixed_zero.bin"),
+    'gpt2': ModelUrlMap(
+        cpp_model_name="gpt2",
+        int4_fixed_zero="https://huggingface.co/kamalojasv/gpt2_int4_fixes_zero/resolve/main/int4_fixed_zero.bin"),
```

Comment:
Can you change it from `kamalojasv/gpt2_int4_fixes_zero` to `kamalojasv/gpt2`?
Also, can you add the CPP quantization script used to quantize this model?